ui/src/views/ResourcePreview.vue

```diff
@@ -27,7 +27,7 @@
 <script lang="ts">
 import { Vue, Component } from 'vue-property-decorator'
 import { namespace } from 'vuex-class'
-import $rdf from '@rdfjs/data-model'
+import { namedNode } from '@rdfjs/data-model'
```

Comment:
I'm curious, does this change have an effect on the amount of code that is imported or is it just a cosmetic preference of yours?

Reply:
Neither really. I followed the warning which said that the module does not have a default export.

Curiously, I actually checked now and it turns out that the typings inside this package are wrong. Bottom line this makes no difference 🙄